app/routes/helpers/yes-no-radios.js

```diff
@@ -9,7 +9,7 @@
  *
  * @return {object} object with `radios` property containing radios component.
  */
-function getYesNoRadios (legendText, id, previousAnswer, errorText = undefined, options = { isPageHeading: true, legendClasses: 'govuk-fieldset__legend--l', inline: true }) {
+function getYesNoRadios (legendText, id, previousAnswer, errorText = undefined, options = { isPageHeading: true, legendClasses: 'govuk-fieldset__legend--l', inline: true }, hintText = '') {
```

Comment:
Rather than adding another parameter to the method, can `hintText` be added to the options object? The JSDoc will also need to be updated accordingly.
The function will need a bit of refactoring to ensure the correct values are used, something like this (in place of line 13)
```js
const inline = options.inline || true
const isPageHeading = options.isPageHeading || true
const legendClasses = options.legendClasses || 'govuk-fieldset__legned--l'
const hintText = options.hintText || ''
```

Also, the test will need to updating to ensure the hint text can be overwritten.

Reply:
yes makes sense thanks